src/components/Combobox/components/DropdownItems/EntityDropdownItem.js

```diff
@@ -9,6 +9,7 @@ export const EntityDropdownItem = ({
   className,
   highlight,
   icon,
+  isDisabled,
```

Comment:
Can you add this prop to the propType definitions in https://github.com/propertybase/react-lds/blob/master/src/components/Combobox/utils/constants.js?